packages/labs/context/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # @lit-labs/context
 
+## 0.1.2
+
+### Patch Changes
+
+- [#2979](https://github.com/lit/lit/pull/2979) [`09156025`](https://github.com/lit/lit/commit/0915602543cd211be19ffd2f54e0082df7ac5ea4) - Fixes `@contextProvider` when multiple instances of an element are created; updates docs to consistently reference `@contextProvided`.
+
+- [#2858](https://github.com/lit/lit/pull/2858) [`5b4edbb6`](https://github.com/lit/lit/commit/5b4edbb6b602f3c40034ebe629b94b2e51ac0c1e) - An element can now provide and request the same context. This lets elements
```

Comment:
Thank you so much for fixing this @augustjk !

Reply:
Thanks for pointing it out in the first place!